lib/pages/devdocs-page.js

```diff
@@ -109,79 +86,4 @@ export default class DevdocsPage extends BaseContainer {
 			'document.querySelector( ".environment-badge" ).style.visibility = "hidden";'
 		);
 	}
-
-	screenshotAllElements( eyes ) {
```

Comment:
This method is used here: https://github.com/Automattic/wp-e2e-tests/blob/master/specs-visdiff/wp-devdocs-visdiff.js.removed#L46

@alisterscott do know why do we keeping these `.removed` viss diff tests?

Reply:
Also here: https://github.com/Automattic/wp-e2e-tests/blob/master/specs-visdiff/wp-devdocs-visdiff.js.removed#L63